### PR TITLE
11.x beta ng13

### PR DESCRIPTION
**Pull request by pschulzk**
Update to Angular 13

### CODE REVIEW

**Review by netwarex — CHANGES_REQUESTED** [review]
I have added some points need to be checked.

Also in local development mode it starts up in production, which should be checked too.

There are a lot's of Foundation Site deprecation error messages, maybe we should make a new ticket on this to investigate it in a hotfix ticket later.

The same could apply to the unit tests, as we should sometime re-enable them.

File: projects/gentics-ui-core/tsconfig.lib.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
     "strictMetadataEmit": true,
     "fullTemplateTypeCheck": true,
     "strictInjectionParameters": true,
-    "enableResourceInlining": true
+    "enableResourceInlining": true,
+    "enableIvy": false
```
[the remark this answers is not in the document]
Comment (pschulzk) [reply projects/gentics-ui-core/tsconfig.lib.json:31]: Yes, there were Ivy issues within consuming dependency structures.

File: projects/gentics-ui-core/package.json [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
   },
   "peerDependencies": {
-    "@angular/common": "^7.2.0 || ^8.0.0",
-    "@angular/core": "^7.2.0 || ^8.0.0",
-    "core-js": "^2.5.4",
-    "tslib": "^1.9.0"
+    "@angular/common": "^13.3.3",
+    "@angular/core": "^13.3.3",
+    "core-js": "^3.6.5",
+    "foundation-sites": "~6.5.3"
```
[the remark this answers is not in the document]
Comment (pschulzk) [reply projects/gentics-ui-core/package.json:19]: Done

File: package.json [hunk 3]
```diff
@@ -59,65 +66,74 @@
...
   "devDependencies": {
-    "@angular-devkit/architect": "~0.13.0",
-    "@angular-devkit/build-angular": "~0.13.0",
-    "@angular-devkit/build-ng-packagr": "~0.13.0",
-    "@angular/cli": "~7.3.3",
-    "@angular/compiler-cli": "~7.2.0",
-    "@angular/language-service": "~7.2.0",
+    "@angular-devkit/architect": "^0.1303.3",
+    "@angular-devkit/build-angular": "^13.3.3",
+    "@angular-devkit/build-ng-packagr": "^0.1002.4",
```
[the remark this answers is not in the document]
Comment (pschulzk) [reply package.json:95]: Cannot identify flaws in build configuration.

File: src/components/modal/modal.service.spec.ts [hunk 4]
```diff
@@ -90,7 +91,7 @@ describe('ModalService:', () => {
             })
         );
 
-        it('displays a title and body',
+        xit('displays a title and body',
```
[the remark this answers is not in the document]
Comment (pschulzk) [reply src/components/modal/modal.service.spec.ts:94]: Re-enabled unit-test.

**Pull request merged**